src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/NomisSyncResourceTest.kt

```diff
@@ -1568,6 +1569,168 @@ class NomisSyncResourceTest : SqsIntegrationTestBase() {
           WhatChanged.ANYTHING,
         )
       }
+
+      @Test
```

Comment:
This test failed with the constraint on. Although I've remove the constraint now as we have some bad data need fixing first.